frontend/common/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -18,8 +18,7 @@ export const Breadcrumbs: React.FunctionComponent<BreadcrumbsProps> = ({
         data-h2-align-items="b(center)"
         href={link.href}
         title={link.title}
-        // eslint-disable-next-line react/no-array-index-key
-        key={index}
+        key={link.title}
```

Comment:
It's because of the fakeExperiences.
```js
const sampleSkill1: Skill = {
  id: faker.datatype.uuid(),
  key: faker.random.word(),
  description: theSkillDescription1,
  name: theSkillString1,
};
```
I'm pretty sure the sample skills like above are getting reused so they have the same id.

Reply:
I guess I'll leave this bug to whoever picks up that issue 😆